networking/src/Hooks/useAPIClients.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
```

Comment:
custom Hook to provide proper sequence for APIClient init. First init AuthAPIClients and then pass those clients down into the other APIClients